evalml/tests/automl_tests/test_automl.py

```diff
@@ -1918,6 +1918,22 @@ class CustomClassificationObjective(BinaryClassificationObjective):
     score_needs_proba = False
     perfect_score = 1.0
     is_bounded_like_percentage = False
+    expected_range = [0, 1]
+    problem_types = [ProblemTypes.BINARY, ProblemTypes.MULTICLASS]
+
+    def objective_function(self, y_true, y_predicted, X=None):
+        """Not implementing since mocked in our tests."""
+
+
+class CustomClassificationObjectiveInf(BinaryClassificationObjective):
+    """Accuracy score for binary and multiclass classification."""
+
+    name = "Classification Accuracy"
+    greater_is_better = True
+    score_needs_proba = False
+    perfect_score = 1.0
+    is_bounded_like_percentage = False
+    expected_range = [float("-inf"), float("inf")]
```

Comment:
Might be good to check a custom objective that has an `expected_range` which is only bounded on one side like `0, inf` but I'm fine if there isn't one